components/devtools/actors/device.rs

```diff
@@ -45,7 +57,10 @@ impl Actor for DeviceActor {
                     from: self.name(),
                     value: SystemInfo {
                         apptype: "servo".to_string(),
-                        platformVersion: "71.0".to_string(),
+                        version: "1.0".to_string(),
```

Comment:
Perhaps this can be `CARGO_PKG_VERSION` since that is what [we use](https://github.com/servo/servo/blob/ff3cd1494ea5c333d8f53450ccd50fe3a287e2a1/ports/servoshell/lib.rs#L185) for `./servo --version`?

Reply:
Good suggestion. Applied.